lms/static/js/fixtures/discovery.html

```diff
@@ -1,26 +1,28 @@
 <section class="courses-container">
-  <div id="discovery-form">
-    <form>
+  <div id="discovery-form" class="wrapper-search-context">
+    <div id="discovery-message" class="search-status-label"></div>
+    <form class="wrapper-search-input">
       <input class="discovery-input" placeholder="Search for a course" type="text"/><!-- removes spacing
       --><button type="submit" class="button postfix discovery-submit" aria-label="Search">
         <i class="icon fa fa-search" aria-hidden="true"></i>
+        <div aria-live="polite" aria-relevant="all">
+          <div id="loading-indicator" class="loading-spinner hidden">
+          <i class="icon fa fa-spinner fa-spin"></i>
+          <span class="sr">Loading</span>
+          </div>
+        </div>
       </button>
     </form>
-    <div id="discovery-message"></div>
-    <div aria-live="polite" aria-relevant="all">
-      <div id="loading-indicator" class="hidden">
-        <i class="icon fa fa-spinner fa-spin"></i> Loading
-      </div>
+    
+    <div id="filter-bar" class="filters hide-phone">
     </div>
-  </div>
 
-  <div id="filter-bar" class="filters hide-phone">
-  </div>
+    <section class="courses">
```

Comment:
thanks for checking @marcotuts.  It doesn't matter to me if its a div or a section.  according to the spec, a section should have a heading element and I don't see one here, so it should probably be a div.  However, what is going to be displayed in this list?  Would there be a benefit for a  screen user to be able to quickly navigate here?  If so, I would add a role="region" to the div and add an aria-label that describes the contents of the div.